### PR TITLE
Add basic documentation to add custom Dashboards block

### PR DESCRIPTION
**Pull request by rchavik**

### CODE REVIEW

File: en/developers/core-plugins/dashboards.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    Path to element file containing the Dashboard html code
+  - column
+    Valid values: ``CroogoDashboard::LEFT``, ``CroogoDashboard::RIGHT``, ``CroogoDashboard::FULL``
+
+Once that is registered, the block should be rendered in the dashboard page.
+
+Displaying Dashboard block contents
+===================================
+
+The registered element simply format and display data in the page. For example,
```
Comment (dakota) [inline en/developers/core-plugins/dashboards.rst]: `The registered element simply formats and displays data on the dashboard page`

File: en/developers/core-plugins/dashboards.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+The following is an excerpt from the element that renders the dashboard block::
+
+    // Plugin/YourPlugin/View/Elements/dashboard/latest_comments.ctp
+    $latestComments = isset($latestComments) ? $latestComments : array();
+    foreach ($latestCmments as $comment):
+        echo $this->Html->para(null, $comment['Comment']['title'];
+    endforeach;
+
+`$latestComments` need to be populated somewhere else. One alternative is to
```
Comment (dakota) [inline en/developers/core-plugins/dashboards.rst]: Should be `needs`

**Comment by smarek** [comment]
Missing keyword `function` in `public startup(Controller $controller)`

**Comment by harvyde** [comment]
it should be 
`public function startup(Controller $controller)`
word function is missing.

**Pull request merged**